server/bin/unittests

```diff
@@ -109,6 +109,7 @@ function _run_allscripts {
     _run pbench-verify-backup-tarballs
     # Invoke pbench-satellite-cleanup in the context of the satellite
     PATH=${_testopt_sat}/unittest-scripts:${_testopt_sat}/bin:${_orig_PATH} _PBENCH_SERVER_CONFIG=${SATCONFIG} _run pbench-satellite-cleanup
+    PYTHONPATH=${_testopt_sat}/unittest-lib:${_orig_PYTHONPATH}
```

Comment:
This is subtle:  the `PATH` definition on line 111 above is actually using `bash`'s "temporary environment variable export" feature, where the updated `PATH` assignment is only enabled for the duration of the `_run` invocation on line 111.

In order for this to work, we want to move this `PYTHONPATH` setting following that `PATH`.

Try this out in `bash` to see how the two variables used by `ls` are only set for the duration of the command:

```
$ unset LS_COLORS
$ unset TIME_STYLE
$ ls -l
$ TIME_STYLE=full-iso LS_COLORS=always ls -l
$ echo "TIME_STYLE='${TIME_STYLE}'; LS_COLORS='${LS_COLORS}'"
$ dircolors    # restores your terminal ls colors
```

Reply:
Good catch; the `PATH` assignment was long enough that I missed that there was a command at the end of the line.

So this `PYTHONPATH` needs to be moved up to 111 along with `PATH` to have effect on `_run`.

Reply:
Maybe break it into three (or more) lines with `\` to escape the unwanted newlines?

```
PATH=...... \
PYTHONPATH=.... \
_PBENCH_SERVER_CONFIG=.... \
command
```

Reply:
I didn't know that. Thank you.  Fixed!